### PR TITLE
feat(component): enhanced dt7 styles to modal, tooltip, card, popover

### PR DESCRIPTION
**Pull request by francisrupert**
## Description

Modal, Tooltip, Card, Popover: CSS-only updates to enhance DT7 alignment.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: docs/components/modal.md [hunk 1]
```diff
@@ -7,27 +7,37 @@ storybook_url: https://vue.dialpad.design/?path=/story/components-modal--default
...
+      </figure>
+    </a>
+    <a class="d-link d-td-none" href="#danger">
+      <figure class="d-m0">
+        <img class="d-bar4 d-w100p" alt="Modal screen: Danger" :src="$withBase('/assets/images/screen-modal--danger.png')">
+        <figcaption>
+          Danger
+        </figcaption>
+      </figure>
+    </a>
```
Comment (braddialpad) [inline docs/components/modal.md:32]: Did we want to change this terminology to "critical" like we did for badge?

File: docs/components/tooltip.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
 <code-well-header bgclass="d-bgc-black-100" class='d-hmn264'>
   <button class="d-btn d-btn--outlined d-tooltip--hover" type="button">
     <div class="d-tooltip d-tooltip__arrow--bottom-center d-ps-absolute">
-      <span>This is a simple tooltip. The tooltip can be positioned in multiple areas too!</span>
+      <span>Simple tooltip</span>
```
Comment (braddialpad) [inline docs/components/tooltip.md:12]: Did we want to change this for the other examples that use this text as well?

File: docs/components/modal.md (hunk 1 above)
Comment (francisrupert) [reply docs/components/modal.md:32]: Ultimately yes, but will hold back for now. Don't want to introduce any kind of breakage or migration requirement for this pure CSS exercise, let alone updating in Vue. t

Next quarter we're rebuilding many key components that use the "Danger" name, including non-trivial overhaul in Vue components.

General, non-comprehensive list of those with 'Danger': button, link, modal, notice, alert, etc...

File: docs/components/tooltip.md (hunk 2 above)
Comment (francisrupert) [reply docs/components/tooltip.md:12]: Actually, I'll clean it up just a tad but don't want to remove how it demos multiple lines. But will leave Vue examples alone.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**